delivery/flux/README.md

```diff
@@ -1,169 +1,205 @@
+## Prerequisites
 
-## Guide
+1. Install `flux` CLI ([official instructions](https://toolkit.fluxcd.io/guides/installation/))
+1. Install Flux's controllers in a cluster: `flux install --version=latest`
 
-This guide is adapted from the [Getting Started][6] guide in the Flux documentation.
+To update the Flux control plane later run `flux install` again.
 
-By the end of this guide you will have
-- Deployed Flux in Cluster
-- Configured Flux to deploy manifests from a Git Repository
+> Alternatively, you may use the `flux bootstrap ...` command to install Flux
+  and store its configuration in a git repo for ongoing reconciliation. To store
+  the configuration in GitHub, first [get a GitHub personal access
+  token](https://github.com/settings/tokens) and set it as the value of
+  environment variable `GITHUB_TOKEN`. An example follows:
 
-### Prerequisites
+```bash
+export GITHUB_TOKEN=<personal_access_token>
+export GITHUB_USER=<github_username>
+export GITHUB_REPO=flux-tests
 
-You will need
-- A Kubernetes Cluster
-- Flux CLI tool. [Installation Instructions][2]
-- A GitHub Personal access token that can create new repositories (Check all permissions under repo). [Instructions][3]
+flux bootstrap github \
+    --owner="${GITHUB_USER}" \
+    --repository="${GITHUB_REPO}" \
+    --private=false \
+    --personal
 
-### Bootstrapping flux
+## verify
+flux get all
+```
+
+## Deliver
+
+You will fork the podtato-head repo and deliver it to your cluster.
+
+### Connect to git repo
 
-Export your GitHub personal access token and username:
+First, [fork the podtato-head repo](https://github.com/podtato-head/podtato-head/fork)
+so that you can add an SSH key to it.
 
+Next, connect this repo to Flux in your cluster by adding an SSH key pair and a
+"source" resource as follows.
+
+The `flux create secret git` command creates an SSH key pair for the specified
+host and puts it into a named Kubernetes secret in Flux's management namespace
+(by default `flux-system`). The command also outputs the public key, which
+should be added to the repos "Deploy keys" in GitHub.
+
+```bash
+GITHUB_USER=<your_github_username>
+flux create secret git podtato-flux-secret --url=ssh://git@github.com/${GITHUB_USER}/podtato-head
 ```
-$ export GITHUB_TOKEN=<your-token>
-$ export GITHUB_USER=<your-username>
+
+If you need to retrieve the public key later you can extract it from the secret as follows:
+
+```bash
+kubectl get secret podtato-flux-secret -n flux-system -ojson | jq -r '.data."identity.pub" | @base64d'
 ```
 
-Run the bootstrap command:
+Use the public key as a Deploy key in your fork of the podtato-head repo. Browse to
+this URL, replacing `<your_github_username>` with your GitHub username:
+`https://github.com/<your_github_username>/podtato-head/settings/keys`. The page will appear as follows:
+
+<img alt="GitHub SSH Deploy Keys" width="400px" src="./images/github-ssh-deploy-keys.png" />
 
+```bash
```

Comment:
Need to add a line here describing that the following snippet is an automated way of adding SSH keys to the repo.

Reply:
Did so in latest commit.